pages/news-and-views/index.vue

```diff
@@ -5,34 +5,16 @@
 				<b>NEWS</b>
 			</h1>
 			<b-row v-for="row in groupedPosts" :key="row.id">
-				<b-col id="api-content" v-for="x in row" :key="x.id">
-					<Post
-						v-if="row.length>1"
-						:picture="x.picture"
-						:title="x.title"
-						:date="x.date"
-						:slug="x.slug"
-					/>
-					<Post
-						v-else
-						:picture="x.picture"
-						:title="x.title"
-						:date="x.date"
-						:slug="x.slug"
-						style="width: 50%"
-					/>
-				</b-col>
+				<b-col id="api-content" v-for="x in row" :key="x.id" />
```

Comment:
Is the `id="api-content"` supposed to be a `class`?

Reply:
Yes, it is. Thanks for the catch!

Reply:
The only reason why I edited this file in the PR was to delete the old `Post` component so it can pass the build.

Reply:
@TeddTech Can you consolidate the work from #70 and your unpushed branch for #64 into this PR? Since they all depend on each other it makes sense to use one pull request.

Reply:
Yes, I agree. Working on it now